infra/common/src/main/java/org/apache/shardingsphere/infra/state/cluster/ClusterStateContext.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.infra.state.cluster;
+
+import lombok.Getter;
+
+/**
+ * Cluster state context.
+ */
+public final class ClusterStateContext {
+    
+    @Getter
+    private ClusterStateType currentState = ClusterStateType.OK;
+    
+    /**
+     * Switch state.
+     * 
+     * @param status status
+     */
+    public void switchState(final ClusterStateType status) {
```

Comment:
It seems that `state` and `status` express the same meaning, can they be named uniformly?

Reply:
I have fixed this, please review again.